### PR TITLE
feat: allow extending cli parser with jcloud cli

### PR DESCRIPTION
**Pull request by alaeddine-13**
**Goal**
- Should allow jina to extend its own CLI with jcloud cli
- [ ] Run [Integration tests GHA](https://github.com/jina-ai/jcloud/actions/workflows/integration-tests.yml) manually & comment the link.

@jina-ai/team-wolf

### CODE REVIEW

File: jcloud/parsers/__init__.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def get_main_parser():
     from .remove import set_remove_parser
 
     # create the top-level parser
-    parser = set_base_parser()
+    parser = set_base_parser(parser=parser)
 
     sp = parser.add_subparsers(
         dest='cli',
```
Comment (hanxiao) [inline jcloud/parsers/__init__.py:17]: this also has to be changed, say `jc_cli`

**Review by hanxiao — CHANGES_REQUESTED** [review]
see comment

File: jcloud/parsers/__init__.py (hunk 1 above)
Comment (alaeddine-13) [reply jcloud/parsers/__init__.py:17]: alright

Comment (alaeddine-13) [reply jcloud/parsers/__init__.py]: done

**Review by hanxiao — APPROVED** [review]

**Pull request merged**